### PR TITLE
Adds Swagger endpoint documentation

### PR DESCRIPTION
**Pull request by mdanish98**
- Documentation URL - http://{host}:{port}/v3/api-docs
- Swagger UI available at http://{host}:{port}/swagger-ui/index.html

Signed-off-by: Mohammad Ghazanfar Ali Danish <ghazanfar.danish@iese.fraunhofer.de>

### CODE REVIEW

File: basyx.common/basyx.http/src/main/java/org/eclipse/digitaltwin/basyx/http/documentation/RepositoryApiDocumentationConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+public class RepositoryApiDocumentationConfiguration {
+
+	private static final String TITLE = "BaSyx Repository";
+	private static final String DESCRIPTION = "BaSyx Components API";
+	protected static final String VERSION = "2.0";
+	private static final String LICENSE = "MIT Licence";
+	private static final String LICENSE_URL = "https://opensource.org/licenses/mit-license.php";
+	private static final String CONTACT_URL = "https://www.eclipse.org/basyx/";
+	private static final String CONTACT_EMAIL = "some-email@iese.fraunhofer.de";
+	private static final String CONTACT_NAME = "Frank Schnicke";
```
Comment (FrankSchnicke) [inline basyx.common/basyx.http/src/main/java/org/eclipse/digitaltwin/basyx/http/documentation/RepositoryApiDocumentationConfiguration.java:49]: Pleae replace with "The BaSyx Developers"

File: basyx.conceptdescriptionrepository/basyx.conceptdescriptionrepository.component/src/main/resources/application.properties [hunk 2]
```diff
@@ -12,4 +12,4 @@ basyx.backend = InMemory
 #spring.data.mongodb.password=password
 
 # Base Path for Spring Boot Actuator
-management.endpoints.web.base-path=/
+management.endpoints.web.base-path=/
```
Comment (FrankSchnicke) [inline basyx.conceptdescriptionrepository/basyx.conceptdescriptionrepository.component/src/main/resources/application.properties:15]: Please revert this file to its original state

**Review by FrankSchnicke — CHANGES_REQUESTED** [review]

File: basyx.aasrepository/basyx.aasrepository.component/src/main/resources/application.properties [hunk 3]
```diff
@@ -17,7 +17,6 @@ basyx.backend = InMemory
 # mqtt.hostname = localhost
 # mqtt.port = 1883
 
-
```
[the remark this answers is not in the document]
Comment (mdanish98) [reply basyx.aasrepository/basyx.aasrepository.component/src/main/resources/application.properties]: Updated in the latest commit

File: basyx.common/basyx.http/src/main/java/org/eclipse/digitaltwin/basyx/http/documentation/RepositoryApiDocumentationConfiguration.java [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+ */
+public class RepositoryApiDocumentationConfiguration {
+
+	private static final String TITLE = "BaSyx Repository";
+	private static final String DESCRIPTION = "BaSyx Components API";
+	protected static final String VERSION = "2.0";
+	private static final String LICENSE = "MIT Licence";
+	private static final String LICENSE_URL = "https://opensource.org/licenses/mit-license.php";
+	private static final String CONTACT_URL = "https://www.eclipse.org/basyx/";
+	private static final String CONTACT_EMAIL = "some-email@iese.fraunhofer.de";
```
[the remark this answers is not in the document]
Comment (mdanish98) [reply basyx.common/basyx.http/src/main/java/org/eclipse/digitaltwin/basyx/http/documentation/RepositoryApiDocumentationConfiguration.java]: Replaced this in the latest commit

File: basyx.common/basyx.http/src/main/java/org/eclipse/digitaltwin/basyx/http/documentation/RepositoryApiDocumentationConfiguration.java (hunk 1 above)
Comment (mdanish98) [reply basyx.common/basyx.http/src/main/java/org/eclipse/digitaltwin/basyx/http/documentation/RepositoryApiDocumentationConfiguration.java]: Replaced this in the latest commit

File: basyx.conceptdescriptionrepository/basyx.conceptdescriptionrepository.component/src/main/resources/application.properties (hunk 2 above)
Comment (mdanish98) [reply basyx.conceptdescriptionrepository/basyx.conceptdescriptionrepository.component/src/main/resources/application.properties]: Updated in the latest commit

**Review by FrankSchnicke — APPROVED** [review]

**Pull request merged**